ws-aspnet/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM cloud9/workspace
+MAINTAINER Peter Bozovic <peter.bozovic@ogilvy.com>
+
+# Getting CoreClr dependencies
+RUN apt-get -y install libunwind8 gettext libssl-dev libcurl3-dev zlib1g libicu-dev
+
+# Installing Kestrel (web server)
+RUN LIBUV_VERSION=1.8.0 \
```

Comment:
Once we update to 16.04 libuv 1.x will be in the repos, will it be possible to change this to use the packaged version without issues at that point? If so, can we put a comment above this so we can remember to remove it in the future.